src/components/MediaCard/MediaCard.test.tsx

```diff
@@ -22,7 +21,7 @@ describe('<MediaCard />', () => {
 			<MediaCard
 				className={customClass}
 				title="What an amazing title!"
-				href="https://viaa.be/"
+				onClick={action('clicked media card')}
```

Comment:
Gaat dit in principe dan direct on render uitvoeren aangezien er geen () => voor de uitvoering staat?

Reply:
nee, blijkbaar returned die action reeds een functie.
maar er waren wel nog enkele andere zaken mis met de action import